mne/viz/_brain/_brain.py

```diff
@@ -2310,6 +2318,89 @@ def add_label(self, label, color=None, alpha=1, scalar_thresh=None,
             self._labels[hemi].append(label)
         self._renderer._update()
 
+    @fill_doc
+    def add_aseg(self, aseg='aparc+aseg', rois=None, colors=None, alpha=0.5,
+                 smooth=0.9, legend=None, **legend_kwargs):
+        """Add an ROI label to the image.
+
+        Parameters
+        ----------
+        aseg : str
+            The anatomical segmentation file. Default ``aparc+aseg``. This may
+            be any anatomical segmentation file in the mri subdirectory of the
+            subject directory from the Freesurfer recon-all.
+        rois : list
+            The regions of interest to plot. See :func:`mne.get_montage_rois`
+            for one way to determine regions of interest. Regions can also be
+            chosen from :ref:`freesurfer lookup table`.
+        colors : list | matplotlib-style color | None
+            A list of anything matplotlib accepts: string, RGB, hex, etc.
+            (default Freesurfer lookup table colors).
+        alpha : float in [0, 1]
+            Alpha level to control opacity.
+        %(smooth)s
+        legend : bool | None
+            Add a legend displaying the names of the ``rois``. Default is
+            ``True`` if the number of ``rois`` is 10 or less.
+        **legend_kwargs : kwargs
+            Keyword arguments to be passed to
+            :meth:`pyvista.BasePlotter.add_legend`.
```

Comment:
Can be more compact as:
```suggestion
        legend : bool | None | dict
            Add a legend displaying the names of the ``rois``. Default (None)
            is ``True`` if the number of ``rois`` is 10 or fewer. Can also be a
            dict of `kwargs` to pass to :meth:`pyvista.BasePlotter.add_legend`.
```